shared/chat/conversation/notices/create-team-notice/container.js

```diff
@@ -23,7 +25,12 @@ const mapDispatchToProps = (dispatch: Dispatch) => ({
     dispatch(
       navigateAppend([
         {
-          props: {conversationIDKey},
+          props: {
+            onCreateNewTeam: name => {
```

Comment:
Just for posterity we talked about not putting closures in the route

Reply:
Done.